microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,145 +20,184 @@ ms.date: 05/04/2021
 
-1. Press `Ctrl`+`P` (Windows, Linux) or `Command`+`P` (macOS).  The **Open File** dialog opens.
+1. In a new window or tab, go to a webpage, such as the [Accessibility-testing features demo](https://microsoftedge.github.io/Demos/devtools-a11y-testing/).  (The source files for that webpage are at [MicrosoftEdge / Demos > devtools-a11y-testing](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-a11y-testing).)
 
-   ![The Open File dialog.](../media/resources-command-menu-empty.msft.png)
+1. To open DevTools, right-click the webpage, and then select **Inspect**. Or press `Ctrl`+ `Shift`+ `I` (Windows, Linux) or `Command`+ `Option`+ `I` (macOS).
 
-1. Select the file from the dropdown list, or start typing the filename and press `Enter` once the correct file is highlighted in the autocomplete box.
+1. When DevTools has focus, press `Ctrl`+ `P` (Windows, Linux) or `Command`+`P` (macOS). Or, you can click the (![Customize icon](../media/customize-devtools-icon-light-theme.png)) menu button in DevTools and then select **Open file**.  The Command Menu opens, displaying the **Open File** list:
 
-   ![Type a filename in the Open File dialog.](../media/resources-command-menu-file-search.msft.png)
+   ![The Command Menu, displaying the Open File list](../media/resources-command-menu-empty.msft.png)
+
+1. If there's a greater-than (>) prompt, press `Backspace` to get to the **Open File** prompt.
+
+1. Start typing the filename, and then press `Enter` when the correct file is highlighted in the autocomplete box, or select the file from the dropdown list:
+
+   ![Typing part of a filename in the Open File list of the Command Menu](../media/resources-command-menu-file-search.msft.png)
 
 
 <!-- ====================================================================== -->
-## Open resources in the Network tool
+## Browse resource files in the Network tool
+
+In the **Network** tool, you can inspect the resource files that make up the current webpage, such as `.html`, `.css`, `.js`, and image files.  To learn how to get the details for a specific resource, see [Inspect the details of the resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
+
+When you go to the **Network** tool and then refresh the page, all the webpage's resource files are listed in the **Network** tool, one row per resource file:
+
+![Webpage resource files listed in the Network Log, one resource file per row](../media/resources-network-resources.msft.png)
 
-See [Inspect the details of a resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
-<!--why?-->
+When **All** is selected as the resources filter, all the retrieved resource files for the current webpage are listed, in the lower left:
 
-<!-- need lead-in -->
+![Inspect a resource in the Network tool](../media/resources-network-response.msft.png)
```

Comment:
Looking at present rendered file, with its present design/flow, pls narrow the red box to only be around "All" button, not Fetch/XHR etc buttons.

I don't recall the context in which I proposed to put red box around all filter buttons. 
Section:
https://github.com/MicrosoftDocs/edge-developer/blob/user/v-danwesley/gs-view-page-resources/microsoft-edge/devtools-guide-chromium/resources/index.md#browse-resource-files-in-the-network-tool
It's the 2nd png in that section has the wide red box.

Bottom relevant section has box around CSS button, and that's the section, if any, where there should be a wide red box on all filters.  d/k if I'll enter comment to do that.
https://github.com/MicrosoftDocs/edge-developer/blob/user/v-danwesley/gs-view-page-resources/microsoft-edge/devtools-guide-chromium/resources/index.md#browse-resource-files-by-file-type-by-filtering-in-the-network-tool

Reply:
should be good now